licenses/com.googlecode.plist_ddplist.txt

```diff
@@ -0,0 +1,20 @@
+dd-plist - An open source library to parse and generate property lists
```

Comment:
Haha, I totally forgot about the `licenses` sub folder... Lat change: 9 years ago...

OK to add something there... We need to cleanup at some stage.

SBOM generation etc.